data/remotes.d/lvfs.conf

```diff
@@ -10,3 +10,8 @@ FirmwareBaseURI=https://fwupd.org/downloads
 AutomaticReports=false
 AutomaticSecurityReports=false
 ApprovalRequired=false
+# Authentication for S3-hosted firmware repositories (not used by LVFS)
+#AuthType=s3
+#AwsAccessKey=your-access-key
+#AwsSecretKey=your-secret-key
+#AwsRegion=us-east-1
```

Comment:
i'd prefer these just to be documented in the manpage; I don't want to confuse the ODMs any more than they already are.  I think using s3 is a pretty niche feature.